backend/onedrive/api/types.go

```diff
@@ -7,7 +7,7 @@ import (
 )
 
 const (
-	timeFormat = `"` + time.RFC3339 + `"`
```

Comment:
I think you are right. But I think that's a problem here, because this constant is used to marshal JSON for the API:

https://github.com/nielash/rclone/blob/f7ed294061bc565ce58de55d584ecbc276b767f2/backend/onedrive/api/types.go#L73-L81

So for OneDrive Personal (which supports mS precision), we would be losing precision in one direction.

Reply:
That's fine. It was just a passing comment!